doc/swss-schema.md

```diff
@@ -620,7 +622,27 @@ Equivalent RedisDB entry:
     12) "0"
     127.0.0.1:6379>
 
-### Configuration files
+---------------------------------------------
+### WARM\_RESTART\_TABLE
+    ;Stores application and orchdameon warm start status
+    ;Status: work in progress
+    key             = WARM_RESTART_TABLE:process_name         ; process_name is unique process identifier
+    restart_count   = 1*10DIGIT                               ; a number between 0 and 2147483647,
+                                                              ; count of warm start times.
+    state           = "init" / "restored" / "synced"
+
+
+## Configuration DB schema
+
+### WARM\_RESTART
+    ;Stores system warm start configuration
+    ;Status: work in progress
+
+    key             = WARM_RESTART:name         ; name is the name of SONiC docker or "system" for global configuration
+    enable          = "true" / "false"          ; default value as false
```

Comment:
timer to determine when to do the reconciliation.

Reply:
I thought we had agreed to remove this docker-specific knob to turn on/off this feature, and make use of a system-wide knob instead, right?

Reply:
I think the agreement is to keep both docker-specific knob and system-wide knob. 
The docker-specific knob will mainly be used by developer.

Reply:
I agree to have both knobs which give convenience .  Can we be more specific in the document how system wide knob work on conjunction with docker knob?

Reply:
Agree, I like the idea of keeping a docker-specific knob for tshooting purposes.

Reply:
The expect behavior:
<1> If system wide knob  is enabled, then docker level know will be ignored. 
There is no point doing system warm reboot while specific docker warm restart is disabled.
<2> If system wide knob  is disabled,  docker level knob will be checked.
The use case is that we may want to docker level upgrade for dockers like bgp and swss.
Will put more info in the schema comment section.